spec/03-plugins/37-opentelemetry/02-schema_spec.lua

```diff
@@ -33,4 +33,16 @@ describe("Plugin: OpenTelemetry (schema)", function()
       }
     }, err)
   end)
+
+  it("accepts variable values", function()
```

Comment:
could you add a test for a failing template as well?

Reply:
I haven't touch original resource attributes schema much, except than adding a compiled template. It is still a map with boolean/string/number type. In the future, we might add validator limiting variable to be existing env vars.

Reply:
I mean, maybe we could add a test that makes the schema validation return the `"value '" .. v .. "' is not in supported format"` error you've defined in the schema. An invalid template should do it.

Reply:
on second thought, perhaps it'd be a good idea to remove the schema validation that enforces a valid template, it could constitute a breaking change for some